### DISCUSSION TITLE
Add sensor (like temperature) to separator card

### DISCUSSION THREAD

**Discussion by wjbeckett** [Clooos/Bubble-Card#397] [Feature requests]
I'd like to see the ability to add a sensor value to the separator, like a temperature value. This would give you the ability to quickly see the temperature of a room for example.

**Comment by bocchinofr** [comment]
it well be a great features...is it possibile ?

**Comment by Clooos** [comment]
I'm working on it and it will be a new feature in the next release 🍻

**Comment by Clooos** [comment]
This now possible with the sub-buttons in the new v2 beta 🎉
